index.js

```diff
@@ -386,7 +386,7 @@
         return x != null && (
           R.type(type.prototype['@@type']) === 'String' ?
             x['@@type'] === type.prototype['@@type'] :
-            R.type(x) === R.nth(1, R.match(/^function (\w*)/, String(type)))
+            R.type(x) === R.nth(1, R.match(/function (\w*)/, String(type)))
```

Comment:
Okay. I'm happy with either approach, in that case. :)

Did you see my comment about changing `S.if` to `S.is` in the commit message and pull request title?